modules/log6x-configuring-otlp-output.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-configuring-otlp-output_{context}"]
+= Configuring OTLP output
+
+The OpenTelemetry Protocol (OTLP) output allows the cluster administrator to collect and forward logs to OTLP receivers. The OTLP output sends data over HTTP using JSON encoding by adhering to the https://opentelemetry.io/docs/specs/otlp/[OpenTelemetry Observability Framework].
+
+:FeatureName: The OTLP output log forwarder
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+* To enable OTLP, create or edit the `ClusterLogForwarder` CR using the template below:
++
+////
+.Example ClusterLogForwarder CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  annotations:
+    observability.openshift.io/tech-preview-otlp-output: "enabled"
+  name: clf-otlp
+spec:
```

Comment:
It should be expected to see in a so much realistic example the ServiceAccount defined as it's a must for running the collectors, in this example, it's missing. 
```
spec:
  serviceAccount:
    name: <service_account_name> 
```

The same is valid for the rest of the examples. It changes how it's in Logging 5.

Also, I'd expect to read a mention that as per PRE requirement have assigned the clusterrole `collect-audit-logs` for being able to collect the audit logs as per https://docs.openshift.com/container-platform/4.16/observability/logging/log_collection_forwarding/log-forwarding.html#log-collection-rbac-permissions_log-forwarding.

Other option could be to delete all lines share below this line for being clear that the example is not valid and it only shows how to create/define the pipeline, but it can't be used for copy&paste directly to create a clusterLogForwarder CR (this option makes more sense to me):
```
apiVersion: observability.openshift.io/v1
kind: ClusterLogForwarder
metadata:
  name: <log_forwarder_name>
  namespace: <log_forwarder_namespace>
```

@libander